### DISCUSSION TITLE
DB_CHARSET and DB_COLLATION

### DISCUSSION THREAD

**Discussion by Stev24** [directus/directus#21142] [Feature Requests]
### Summary

At the moment, Directus has an option to set DB_CHARSET. But it is not possible to set the collation. 
I suggest to make a variable like DB_COLLATION to be able to set collation.

### Motivation

Being able to set the right collation for the database.

### Detailed Design

no design

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

..

### Alternatives

..

### Adoption Strategy

..

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
